config/samples/dataplane_v1beta1_openstackdataplane_customnetworks.yaml

```diff
@@ -0,0 +1,174 @@
+          template: |
+            ---
+            network_config:
+            - type: interface
+              name: nic2
+              mtu: 1500
+              addresses:
+                - ip_netmask:
+                    {{ ctlplane_ip }}/{{ ctlplane_subnet_cidr }}
+            - type: ovs_bridge
+              name: {{ neutron_physical_bridge_name }}
+              mtu: 1500
+              use_dhcp: false
+              dns_servers: {{ ctlplane_dns_nameservers }}
+              domain: []
+              addresses:
+              - ip_netmask: {{ lookup('vars', networks_lower["External"] ~ '_ip') }}/{{ lookup('vars', networks_lower["External"] ~ '_cidr') }}
+              routes: [{'ip_netmask': '0.0.0.0/0', 'next_hop': '192.168.1.254'}]
+              members:
+              - type: interface
+                name: nic1
+                mtu: 1500
+                # force the MAC address of the bridge to this interface
+                primary: true
+              - type: vlan
+                mtu: 1500
+                vlan_id: 20
+                addresses:
+                - ip_netmask:
+                    172.17.0.101/24
+                routes: []
+              - type: vlan
+                mtu: 1500
+                vlan_id: 25
+                addresses:
+                - ip_netmask:
+                    172.18.0.101/24
+                routes: []
+              - type: vlan
+                mtu: 1500
+                vlan_id: 22
+                addresses:
+                - ip_netmask:
+                    172.10.0.101/24
+                routes: []
+        ansibleVars: |
+          # edpm_network_config
+          # Default nic config template for a EDPM compute node
+          # These vars are edpm_network_config role vars
+          edpm_network_config_template: /runner/network/nic-config-template
+          edpm_network_config_hide_sensitive_logs: false
+          edpm_network_config_update: false
+          #
+          # These vars are for the network config templates themselves and are
+          # considered EDPM network defaults.
+          neutron_physical_bridge_name: br-ex
+          neutron_public_interface_name: enp7s0
+          ctlplane_mtu: 1500
+          ctlplane_subnet_cidr: 24
+          ctlplane_gateway_ip: 192.168.1.254
+          ctlplane_host_routes:
+          - ip_netmask: 0.0.0.0/0
+            next_hop: 192.168.1.254
+          external_mtu: 1500
+          external_vlan_id: 4
+          external_cidr: '24'
+          external_host_routes: []
+          internal_api_mtu: 1500
+          internal_api_vlan_id: 20
+          internal_api_cidr: '24'
+          internal_api_host_routes: []
+          storage_mtu: 1500
+          storage_vlan_id: 21
+          storage_cidr: '24'
+          storage_host_routes: []
+          tenant_mtu: 1500
+          tenant_vlan_id: 22
+          tenant_cidr: '24'
+          tenant_host_routes: []
+          role_networks:
+          - InternalApi
```

Comment:
This is off-topic: We no longer need these network names in heat resources and can probably get rid of the redundant conversion using networks_lower, though the final var structure would be dependent on what we get from baremetal provisioning.